run-tests.js

```diff
@@ -275,7 +264,7 @@ const testExchange = async (exchange) => {
     if (failed) {
         logMessage+= 'FAIL'.red;
     } else if (hasWarnings) {
-        logMessage = (warnings.length ? warnings.join (' ') : 'WARN').yellow;
+        logMessage = 'WARN'.yellow + (warnings.length ? warnings.join (' ') : 'NO_OUTPUT');
```

Comment:
this was not perfect before change, now the first word will always be `WARN` (yellow) independently what is the `warnings` messages.